pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -13,6 +13,60 @@ import (
 	"github.com/holiman/uint256"
 )
 
+// Function to create the getHighLen Hinter
+func newGetHighLenHint() hinter.Hinter {
+	return &hinter.GenericZeroHinter{
+		Name: "getHighLen",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			scalarU, err := hinter.GetVariableAs[int](&ctx.ScopeManager, "scalar_u.d2")
+			if err != nil {
+				return fmt.Errorf("failed to resolve scalar_u.d2: %w", err)
+			}
+
+			scalarV, err := hinter.GetVariableAs[int](&ctx.ScopeManager, "scalar_v.d2")
+			if err != nil {
+				return fmt.Errorf("failed to resolve scalar_v.d2: %w", err)
+			}
+
+			// Calculate the highest bit length
+			lenHi := max(bitLength(scalarU), bitLength(scalarV)) - 1
+
+			// Assign the result to ids.len_hi
+			if err := ctx.ScopeManager.AssignVariable("len_hi", lenHi); err != nil {
+				return fmt.Errorf("failed to assign len_hi: %w", err)
+			}
+
+			return nil
+		},
+	}
+}
+
+// Function to create the hinter with the resolver
+func createGetHighLenHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
```

Comment:
you need to fetch `len_hi`, `scalar_u` and `scalar_v` here, using method `resolver.GetResOperander("reference_name")` and pass then to `newGetHighLenHint()`. [Reference](https://github.com/NethermindEth/cairo-vm-go/blob/75bf4e33cdc073f2d1b277c051506dcd2bc9b573/pkg/hintrunner/zero/zerohint_others.go#L185)